frontend/src/router.tsx

```diff
@@ -0,0 +1,38 @@
+import { Navigate, useRoutes } from "react-router";
+import Login from "./components/Login/login.tsx";
```

Comment:
import ide ovako: import Login from './components/Login/Login.tsx';
a ne login.tsx
Plus u folderu od login-a i register-a imas index iz kojeg exportas a nigdi ne korists taj export nego iz glavnog file-a 
Znaci to je ili nepotrebno, ili ga importat iz te lokacije, iako po trenutnoj sttrukturi nije potriban jer imas samo jedan file, to se inace koristi kad u istom folder-u imas vise file-ova pa da su ti svi import-i s iste lokacije, a ne iz 5 razlicitih